kernel/kernel-api/src/main/java/io/delta/kernel/internal/actions/CommitInfo.java

```diff
@@ -189,18 +191,23 @@ public static long getRequiredInCommitTimestamp(
     public static Optional<CommitInfo> getCommitInfoOpt(
             Engine engine,
             Path logPath,
-            long version) throws IOException {
```

Comment:
This was never thrown here. Also removed corresponding handling in `SnapshotImpl`.